labs/advanced-topics/start.sh

```diff
@@ -9,13 +13,14 @@ while ! nc -z kafka 9092; do
 done 
 echo "Kafka is now ready!"
 
-kafka-topics --bootstrap-server kafka:9092 \
+kafka-topics --create \
+    --bootstrap-server kafka:9092 \
     --topic vehicle-positions \
-    --create \
     --partitions 6 \
     --replication-factor 1
 
 docker container run -d \
     --name producer \
     --net advanced-topics_confluent \
+    --platform linux/amd64 \
```

Comment:
This suppresses the following warning on macOS Apple Silicon:
```
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```

Reply:
This is the only container for which I could not find an arm64 compatible version. On an Apple Silicon Mac the performance is adequate to run this single container in emulation mode, now that all other containers can run natively on Apple Silicon (see docker compose file changes in this PR).